libraries/psibase/native/src/Watchdog.cpp

```diff
@@ -71,10 +71,14 @@ namespace psibase
          elapsedOrStart(CpuClock::now().time_since_epoch()),
          handler(handler)
    {
+#ifdef __APPLE__
+      cpuclock = CLOCK_UPTIME_RAW;
```

Comment:
`CpuClock::now` has a default parameter of `CLOCK_THREAD_CPUTIME_ID`. Either change that default to match this, or remove the default and change all uses of `now` to `CpuClock::now(cpuclock)`.

Reply:
they now match with a value that works by default on APPLE